src/modules/commentPopup.js

```diff
@@ -0,0 +1,130 @@
+import { makeComment } from './apis.js';
+
+// get comment count from local storage
+const count = JSON.parse(localStorage.getItem('commentCounter'));
+
+// count the comments💬💬 of each rocket🚀
+const commentCount = async () => JSON.parse(localStorage.getItem('commentCounter'));
+
+// function render data of the homepage
+const popupView = async (
+  rocketId,
+  rocketImage,
+  rocketName,
+  successRate,
+  costPerLaunch,
+  weightMass,
+  rocketType,
+  commentData,
+) => {
+  // Retrieve comment count asynchronously
+  const commentCountValue = await commentCount();
+
+  const popupsContainer = document.querySelector('.popups-container');
+  let popup = '';
+  popup += `
+    <section class="each-popup">
+      <div class='pop-container'>
+        <section class='image-and-close-button'>
+          <i class="fas fa-times-circle" id="modal-btn"></i> 
+          <img src=${rocketImage} alt="" class="popup-item-image">
+        </section>
+
+        <section class='rocket-header-container'>
+          <div class='rocket-description'>${rocketName}</div>
+        </section>
+
+        <section class='rocket-details'>
+          <div class='items-properties'>
+            <div class='items-properties each-property'>Success rate: ${successRate}</div>
+            <div class='items-properties each-property'> Weight: ${weightMass}</div>
+          </div>
+          <div class='items-properties'>
+            <div class='items-properties each-property'>Rocket type: ${rocketType}</div>
+            <div class='items-properties each-property'>Cost per launch: ${costPerLaunch}</div>
+          </div>
+        </section>
+
+        <section class='rocket-header-container'>
+          <div class='rocket-description comments'>comments: ${commentCountValue}</div>
+        </section>
+
+        <section class='message-details'>
+          ${
+  /* eslint-disable camelcase */
```

Comment:
- [ ] Please avoid disabling linter rules as they serve as a way to provide a consistent code format and ensure you use best practices. Instead, consider fixing the errors! 🤓 🔧

Reply:
All the camel cases. I had to disable because I'm destructuring the variables from the endpoint. so how can I tweek these variable names? Please do you know this concept?